app/src/main/kotlin/com/wire/android/notification/broadcastreceivers/ConnectionRequestNotificationDismissReceiver.kt

```diff
@@ -49,14 +51,7 @@ class ConnectionRequestNotificationDismissReceiver(
                 }
 
             sessionScope?.let { scope ->
-                requesterUserId?.let { userId ->
-                    qualifiedIdMapper.fromStringToQualifiedID(userId).let {
-                        scope.conversations.markConnectionRequestAsNotified(it)
```

Comment:
why aren't we marking connection request as notified anymore? 🤔

Reply:
We do, it was just duplicated by mistake 😄 
<img width="1236" alt="image" src="https://user-images.githubusercontent.com/30429749/182637422-6ebfdc46-62ff-495a-a7f5-31bba2a2ed12.png">

Reply:
ooooooh, didn't realize that! 😓 my bad